tests/acceptance/sentry_plugins/test_amazon_sqs.py

```diff
@@ -1,8 +1,8 @@
 from sentry.testutils.cases import AcceptanceTestCase
-from sentry.testutils.silo import region_silo_test
+from sentry.testutils.silo import no_silo_test
 
 
-@region_silo_test(stable=True)
+@no_silo_test(stable=True)
```

Comment:
What does it mean to annotate a test with `no_silo_test` now? Just that tests are broken unless running in monolith mode?

Reply:
Yes, and or more specifically that it is fine not to invoke the full RPC stack.